### PR TITLE
Feature/cv Oriol Escolar

### PR DESCRIPTION
**Pull request by Ury18**

### CODE REVIEW

File: staff/oriol-escolar/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <li>C#</li>
+
+</ul>
+
+</section>
+
+<!-- Languages -->
+<section>
+
+<h2><u>Languages</u></h2>
```
Comment (lexbarba) [inline staff/oriol-escolar/cv/index.html]: The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).

File: staff/oriol-escolar/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+<li>English - First Certificate</li>
+
+</ul>
+
+</section>
+
+<!-- Contact info -->
+<section>
+
+<h2><u>Contact Information</u></h2>
```
Comment (lexbarba) [inline staff/oriol-escolar/cv/index.html]: The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).

File: staff/oriol-escolar/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+<body>
+    
+
+<h1>Oriol Escolar</h1>
+<img src="" alt="Image sample">
+
+<!-- EDUCATION -->
+<section>
+ 
+<h2><u>Education</u></h2>
```
Comment (lexbarba) [inline staff/oriol-escolar/cv/index.html]: The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).

Comment (Ury18) [reply staff/oriol-escolar/cv/index.html]: You are right, removed <u> in the code.

File: staff/oriol-escolar/cv/index.html (hunk 2 above)
Comment (Ury18) [reply staff/oriol-escolar/cv/index.html]: You are right, removed in the code.

File: staff/oriol-escolar/cv/index.html (hunk 1 above)
Comment (Ury18) [reply staff/oriol-escolar/cv/index.html]: You are right, removed in the code.

**Pull request merged**